src/web/mjs/connectors/Alphapolis.mjs

```diff
@@ -43,11 +43,20 @@ export default class Alphapolis extends Connector {
-        `;
         const request = new Request(new URL(chapter.id, this.url), this.requestOptions);
-        return await Engine.Request.fetchUI(request, script);
+        const data = await this.fetchDOM(request, 'viewer-manga-horizontal');
+        try {
+            const pages = JSON.parse(data[0].getAttribute('v-bind:pages'));
+            let pagesList = [];
+            pages.forEach(page => {
```

Comment:
isn't this the same as an array filter ? you could merge both filter in the same one

Reply:
Solved